app/updatemanager.go

```diff
@@ -396,6 +397,33 @@ func (c *ControlMapPool) QueryAndUpdate(state string) (action string) {
 	return "continue"
 }
 
+// TODO - rename to NextUpdateControlMapTime / or similar
+func (c *ControlMapPool) NextControlMapHalfTime(ID string) time.Time {
```

Comment:
Will you rename? I think the currently used name is better, actually...

Reply:
Oh, really? haha, OK, fixed!